proto/iotics/api/interest.proto

```diff
@@ -94,16 +92,13 @@ message Interest {
   message FollowedFeed {
     // Followed feed identifier
     Feed feed = 1;
-    // Feed remote host identifier (If not specified, the Interest is taken to be in scope of the host from which a request is made.)
-    HostID hostId = 2;
   }
 
   // Follower twin unique identifier.
-  TwinID followerTwinId = 2;
+  Twin followerTwin = 2;
```

Comment:
so - follower is ALWAYS on the local host
should we use Twin or just the twinDiD ? 
what is the significance of Twin here otherwise

Reply:
as mentioned above, all responses will return this TwinId object, so it easily be passed to subsequent requests as is.